source/extensions/common/wasm/wasm.h

```diff
@@ -504,7 +504,7 @@ class Wasm : public Envoy::Server::Wasm,
 public:
   Wasm(absl::string_view vm, absl::string_view vm_id, absl::string_view vm_configuration,
        PluginSharedPtr plugin, Upstream::ClusterManager& cluster_manager,
-       Event::Dispatcher& dispatcher);
+       Event::Dispatcher& dispatcher, std::shared_ptr<Stats::StatNameSet> stat_name_set = nullptr);
```

Comment:
Nitpicking now, since this is going to be a problem when upstreaming anyway:
1) `std::shared_ptr<Stats::StatNameSet>` -> `Stats::StatNameSetSharedPtr`.
2) I don't think there is a convention of setting default argument values in prototypes in Envoy.

Reply:
Done (SharedPtr).

Fixed the default argument.  PTAL